src/Sentry/Internal/Http/RetryAfterHandler.cs

```diff
@@ -19,6 +19,7 @@ internal class RetryAfterHandler : DelegatingHandler
         private readonly ISystemClock _clock;
 
         private const HttpStatusCode TooManyRequests = (HttpStatusCode)429;
+        internal static readonly TimeSpan DefaultRetryAfterDelay = TimeSpan.FromSeconds(60);
```

Comment:
`TimeSpan.FromMinutes(1)`

Reply:
Thanks, but I'd rather keep it in terms of seconds, to match the spec.